cmd/vclusterctl/cmd/pause.go

```diff
@@ -56,7 +56,7 @@ vcluster pause test --namespace test
 		},
 	}
 
-	cobraCmd.Flags().StringVar(&cmd.Manager, "manager", "", "The manager to use for managing the virtual cluster, can be either helm or platform.")
+	cobraCmd.Flags().StringVar(&cmd.Driver, "driver", "", "The driver to use for managing the virtual cluster, can be either helm or platform.")
```

Comment:
Same as above, can ignore if this is overkill.
```suggestion
	cobraCmd.Flags().StringVar(&cmd.Driver, "driver", "", "The driver for the virtual cluster, can be either helm or platform.")
```